app/generator/content/get-help-info.js

```diff
@@ -1,7 +1,7 @@
-const getHelpInfo = () => {
+const getHelpInfo = (partLabel) => {

```

Comment:
Is partLabel the best name for this or could this be something more descriptive to the function? Would using moreInformation be more understandable.